deepchecks/vision/checks/distribution/train_test_label_drift.py

```diff
@@ -69,82 +104,35 @@ def run_logic(self, context: Context) -> CheckResult:
         task_type = train_dataset.task_type
         displays = []
 
-        if task_type == TaskType.CLASSIFICATION:
-
-            train_label_distribution = train_dataset.get_samples_per_class()
-            test_label_distribution = test_dataset.get_samples_per_class()
-
-            drift_score, method, display = calc_drift_and_plot(
-                train_distribution=train_label_distribution,
-                test_distribution=test_label_distribution,
-                plot_title='Class',
-                column_type='categorical',
-            )
-
-            values_dict = {'Drift score': drift_score, 'Method': method}
-            displays.append(display)
-
+        if self.alternative_label_transformers is not None:
```

Comment:
use validate_shared_label() (also validate label?)

Reply:
No need because of Context